6212 АльсимовД.П. Лаб.2 Вар.21

Alsik21 opened the pull request:
ФИО: Альсимов Дмитрий
Номер группы: 6212
Номер лабораторной: 2
Номер варианта: 21
Краткое описание решаемой задачи: скачать аудио файлы длиной более 10 секунд.
Краткое описание работы кода: для запуска через командную строку нужно использовать:
1. pip install requests beautifulsoup4 lxml
2. python audio_downloader.py --download-dir "./audio_files" --annotation-file "./annotation.csv" --num-files 60 --use-iterator
 Так же я добавил результаты работы кода

Alsik21 committed:
Add files via upload

Alsik21 committed:
Add files via upload

Alsik21 committed:
Delete lab_2/lab_2.py

IlyaOv reviewed (commented):
Я правильно понял, что звуки в количестве 60 штук так и не скачались?
И не увидел проверки на то, что длина трека больше 10 секунд.

Alsik21 committed:
Исправления

Я исправил то что Вы сказали но теперь код выдаёт то что не нашёл аудиофайлы длиной более 10 секунд. Я пробовал расширить поиск. но это привело к тому что код искал аудио на 372 страницах сайта...
так же нужно скачать ещё одну библиотеку 
pip install mutagen

Alsik21 committed:
Исправления

Исправил ошибки и уменьшил объём кода

Alsik21 committed:
изменения

Alsik21 commented:
в последнем коммите, вынес headers как отдельный метод

IlyaOv reviewed (approved):
Зачтено
